pkg/resources/password_policy_acceptance_test.go

```diff
@@ -180,3 +184,49 @@ func TestAcc_PasswordPolicyMaxAgeDays(t *testing.T) {
 		},
 	})
 }
+
+func TestAcc_PasswordPolicy_migrateFromVersion_0_94_1(t *testing.T) {
```

Comment:
could you check if those tests are working without state upgrader? If they're working, remove state upgraders, but leave the tests

Reply:
Removed state upgraders, left tests.